v3/integration/config.json

```diff
@@ -770,6 +770,9 @@
     },
     "w_subject_given_name_recommended_max_length": {},
     "w_subject_surname_recommended_max_length": {},
-    "w_tls_server_cert_valid_time_longer_than_397_days": {}
+    "w_tls_server_cert_valid_time_longer_than_397_days": {},
+    "e_no_underscores_before_1_6_2": {
+      "ErrCount": 340
```

Comment:
Do you have some examples just to double check (e.g. we should be able to find these in Moz incident reports)

Reply:
Ah, certainly. Better than some examples here is the full dataset (first with only a list of fingerprints, and second with all pem data and openssl textual outputs).

[failedFingerprints.txt](https://github.com/zmap/zlint/files/8063337/failedFingerprints.txt)
[failedPems.pem](https://github.com/zmap/zlint/files/8063341/failures.txt)

Here are some smoke checks from crt.sh

https://crt.sh/?q=005155a41690d7ce8ad69556f5b2aafea595298e0985a2a70e7210ac426fd5c4
https://crt.sh/?q=018e2a96f6ad8d517e001afd4a0d5d700d9fd7093e9c8349bd7591139504fcad
https://crt.sh/?q=e6769a8ebd3b9801aa4abb329880fdd1da1bb2002ec070ed97e267ee34bf3337